WebHostLib/static/assets/player-settings.js

```diff
@@ -294,23 +294,25 @@ const buildOptionsTable = (settings, romOpts = false) => {
   return table;
 };
 
-const toggleRandomize = (event, inputElements) => {
+const toggleRandomize = (event, inputElement, optionalSelectElement) => {
```

Comment:
It is preferable to use default values of null for optional arguments. Consider changing this to:
```js
const toggleRandomize = (event, inputElement, selectElement = null) =>
```